src/data/mutations.graphql

```diff
@@ -23,3 +23,22 @@ mutation EditUser($input: EditUserInput!) {
     id
   }
 }
+
+mutation CreateColony($input: CreateColonyInput!) {
+  createColony(input: $input) {
+    id
+    colonyAddress
+    colonyName
+    avatarHash
+    description
+    displayName
+    guideline
+    website
+  }
+}
+
+mutation EditColonyProfile($input: EditColonyProfileInput!) {
+  editColonyProfile(input: $input) {
```

Comment:
Can you return all the fields this mutation can possibly change (alongside the `id`)? Then the Apollo cache will automatically pick it up.

So that's probably `avatarHash`, `description`, `displayName`, `guideline`, `website`